pyne/openmc_utils.py

```diff
@@ -6,17 +6,33 @@
 import sys
 from warnings import warn
 from collections import namedtuple
+import numpy as np
+import tables as tb
 
 if sys.version_info[0] == 2:
     from HTMLParser import HTMLParser
 else:
     from html.parser import HTMLParser
 
+from pyne.utils import QAWarning
+from pyne.mesh import MeshTally, HAVE_PYMOAB
+if HAVE_PYMOAB:
+    from pyne.mesh import NativeMeshTag
+else:
+    warn("The PyMOAB optional dependency could not be imported. "
+         "Some aspects of the openmc module may be incomplete.",
+         QAWarning)
 from pyne import nucname
 from pyne.utils import QAWarning
 
 warn(__name__ + " is not yet QA compliant.", QAWarning)
 
+try:
+    import openmc
+except:
+    warn("The openmc (OpenMC Python API) could not be imported. "
```

Comment:
Should this be more than a warning? Is there any remaining functionality in this module if we can’t import `OpenMC`?

Reply:
Yes, there are. For example, the class `AceTable` is not dependent on OpenMC Python API.